pymongo/errors.py

```diff
@@ -114,7 +114,7 @@ class NotMasterError(AutoReconnect):
     Subclass of :exc:`~pymongo.errors.AutoReconnect`.
     """
     def __str__(self):
```

Comment:
As discussed, can you add a test that asserts that the "full error" string is included in str(exc) and [traceback.format_exc()](https://docs.python.org/3/library/traceback.html#traceback.format_exc?

Reply:
Done.